src/pages/example--gift-card.mdx

```diff
@@ -0,0 +1,80 @@
+import { Callout } from "nextra-theme-docs";
+
+# Gift Card
+
+Let's build a UI to send and redeem a gift card using smart contracts on Cardano.
+
+## Covered in this tutorial
+
+- writing `Aiken`
+- parameterizing scripts
+- using [lucid](https://lucid.spacebudz.io/)
+- using [deno fresh](https://fresh.deno.dev/)
+
+<Callout type="info">
+  ★ You should install deno using these
+  [instructions](https://deno.land/manual@v1.29.1/getting_started/installation).
+</Callout>
+
+## What is a gift card?
+
+In the context of this tutorial a gift card will involve locking some assets
+in a smart contract. While some assets are being locked, we'll mint an NFT in
+the same transaction. This NFT could be sent anywhere and the owner of the NFT
+can burn it to unlock the assets there were previously locked. We can consider
+the NFT as a gift card.
+
+## Aiken is the easy part
+
+Let's go ahead and create a new `Aiken` project:
+
+```sh
+aiken new my-org/gift-card
+cd gift-card
+```
+
+<Callout type="info">
+  `my-org` above can be replaced by any name. We recommend using the name of a
+  Github organization or your own username.
+</Callout>
+
+We've already covered what `aiken new` generates in a previous tutorial so
+let's jump right into some code.
+
+Go ahead and remove the `lib/` folder, we won't be needing that for this tutorial.
+
+```
+rm -rf lib
+```
+
+Now let's create a new file in the `validators/` folder called `main.ak`.
+
+```
+touch validators/main.ak
+```
+
+<Callout type="info">
+  `main.ak` could be named anything. Any file in `validators/` is allowed to
+  export as many validators as you'd like.
+</Callout>
+
+Now let's open the project folder in our favorite editor and define two empty
+validator functions.
+
+```aiken filename="validators/main.ak"
+use aiken/transaction.{ScriptContext}
+
+validator {
+  fn gift_card(rdmr, ctx: ScriptContext) -> Bool {
+    todo @"mint and burn"
+  }
+}
+
+validator {
+  fn redeem(d, r, ctx: ScriptContext) -> Bool {
+    todo @"redeem"
+  }
+}
```

Comment:
What about using this example to demonstrate a combined validator?

Reply:
@KtorZ I thought about it and I don't want to redo the corresponding example in the main aiken repo.

So I decided we'll just make a new example with the multi-validator and then follow that up with it's own tutorial.